content/material/2019-11-15-intro-dplyr.Rmd

```diff
@@ -347,33 +353,29 @@ economics_data <- economics %>%
 economics_data
 ```
 
+-Explore the variables in the dataset by querying the economics dataframe (`?economics`)
+
+-Using what you have learned today try to:
+1. Create a new variable called `unemploy_rate` that calculates the unemployment rate (i.e. proportion of the population that is unemployed)
+2. For the same decade, calculate the maximum, minimum and mean values for duration of unemployment and unemployment rate per year, then arrange these values in descending order by year
 
 ```{r, echo=FALSE}
 economics_data %>% 
     filter(year >= 1990, year <= 2000) %>% 
     mutate(unemploy_rate = unemploy / pop) %>% 
     select(year, unemploy_rate, uempmed) %>% 
-    gather(variable, value, -year) %>% 
-    group_by(year, variable) %>% 
+    #gather(variable, value, -year) %>% 
```

Comment:
delete this line since we aren't covering it.

Reply:
```suggestion
```